Jenkinsfile.cicd

```diff
@@ -435,11 +408,14 @@ def deployStage(String stageEnv, String projectEnv, String hostEnv, String pathE
         Backend: {
           echo "Processing DeploymentConfig ${APP_NAME}-${JOB_NAME}-backend..."
           def dcBackendTemplate = openshift.process('-f',
-            'openshift/backend.dc.yaml',
-            "REPO_NAME=${REPO_NAME}",
-            "JOB_NAME=${JOB_NAME}",
+            "${EMAIL_MICROSRV_DC}",
+            "APP_LABEL=${EMAIL_MICROSRV_APP_LABEL}",
+            "IMAGE_NAME=${EMAIL_MICROSRV_IMAGE_NAME}",
             "NAMESPACE=${projectEnv}",
-            "APP_NAME=${APP_NAME}"
+            "SECRET_NAME=cmsg-client",
+            "CONFIG_MAP_NAME=cmsg-config",
+            "CMSG_SENDER=NR.CommonServiceShowcase@gov.bc.ca",
```

Comment:
Don't need to fix anything, but as a note, using single quote strings for things that do not need interpolation is the "groovy" way of handling strings, whereas things with templating and need interpolation need to use double quotes.